tests/test_parsers.py

```diff
@@ -273,6 +274,21 @@ def test_parse_sssom_json(self):
             f"{self.json_file} has the wrong number of mappings.",
         )
 
+    def test_split_msdf(self):
```

Comment:
Thank you for the tests!!!

Reply:
I realized that the test for `split` in the CLI only checked if the command ran without error, without checking any output :face_with_spiral_eyes:

Reply:
Don’t get me on a rant about the cli testing here…